Resources/public/js/views/services/plugins/ez-locationsearchplugin.js

```diff
@@ -49,31 +49,55 @@ YUI.add('ez-locationsearchplugin', function (Y) {
             // not yet supported by the REST API
             // see eZ.SubItemListView and https://jira.ez.no/browse/EZP-24315
             // query.body.ViewInput.LocationQuery.SortClauses = e.search.sortClauses;
-            
+
             contentService.createView(query, Y.bind(function (error, result) {
                 var attrs = {'loadingError': true};
 
                 if ( !error ) {
+                    var parsedResult = this._parseSearchResult(result);
+
                     attrs.loadingError = false;
-                    attrs[e.resultAttribute] = this._parseSearchResult(result);
+                    if (e.resultTotalCountAttribute) {
```

Comment:
Since we are starting to have a few properties in this even, could we document it in the JSdoc ?